server/concepts/Tournament.ts

```diff
@@ -123,6 +123,27 @@ export default class Tournament implements TournamentWithoutMS {
     /* eslint-enable @typescript-eslint/no-non-null-assertion */
   }
 
+  public static async getById(tournamenId: string, token: string) {
+    const tournament = await TournamentRepository.getById(tournamenId);
+    if (!tournament) return null;
+    const enrichedTournament = await Tournament.enrichWithMSUsers(tournament, token);
+    return enrichedTournament;
```

Comment:
```suggestion
    return Tournament.enrichWithMSUsers(tournament, token);
```

Reply:
are you sure? because i think it will return Promise

Reply:
`return` resolving Promise @PrzemekRychter

Reply:
ok